src/ml_tooling/plots/target_feature_distribution.py

```diff
@@ -0,0 +1,63 @@
+from matplotlib import pyplot as plt
+from matplotlib.axes import Axes
+import numpy as np
+
+from ml_tooling.utils import DataType
+from ml_tooling.plots.utils import _plot_barh
+
+
+def plot_target_feature_distribution(
+    target: DataType,
+    feature: DataType,
+    title: str = "Target feature distribution",
+    method: str = "mean",
+    ax: plt.Axes = None,
+) -> Axes:
+    """
+    Creates a plot which compares the mean or median
+    of a binary target based on the given category features.
+    Using np.nanmean or np.nanmedian
```

Comment:
This is an implementation detail  - more appropriate to inform the user that `NaN`s are ignored in the calculation